### DISCUSSION TITLE
Mute/UnMute WebRTC stream While playing in WebRTC iOS SDK

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#6165] [Show and tell]
Let me explain how to mute and unmute the incoming stream.

- Let's assume that you're publishing a live stream to Ant Media Server with a stream id `stream1` 
- Let's assume again that you're playing that stream with WebRTC-iOS-SDK as explained in this [blog post at Step 4:Play WebRTC Live Stream in iOS][1] 
- Call the following method to mute the playing stream after live stream has started
```swift
  self.antMediaClient?.enableAudioTrack(trackId: "stream1", enabled: false);
```

You can give `true` in the second parameter to unmute the audio track again.

You can use the `enableVideoTrack` in the same way. The good thing is that when you call these methods, you make server to enable/disable sending streams. It means that you can decrease network usage if it's important for you.

PS: This question is first asked and answered in [stackoverflow](https://stackoverflow.com/questions/78119529/mute-unmute-webrtc-stream-while-playing-in-ant-medias-webrtc-ios-sdk/78119610#78119610)
Cheers

  
  [1]: https://antmedia.io/build-and-try-webrtc-ios-app-for-dummies/

**Comment by huynguyen96419** [comment]
How to mute/unmute in WebRTC Android SDK?

**Reply by zerostack0** [reply]
Hi @huynguyen96419 

You can use the function below:

`webRTCClient.setAudioEnabled(false);`
